pkg/core/engine/main.go

```diff
@@ -126,43 +127,67 @@ func (e *Engine) Prepare() (err error) {
 	}
 
 	err = e.LoadConfigurations()
-	if err != nil {
-		return err
+
+	if len(e.Pipelines) == 0 {
+		logrus.Errorln(err)
+		return fmt.Errorf("no valid pipeline found")
 	}
 
-	err = e.InitSCM()
+	// Don't exit if we identify at least one valid pipeline configuration
 	if err != nil {
-		return err
+		logrus.Errorln(err)
+		logrus.Infof("\n%d pipeline(s) successfully loaded\n", len(e.Pipelines))
 	}
 
+	// If one git clone fails then we exit
+	err = e.InitSCM()
+
 	return err
 }
 
 // ReadConfigurations read every strategies configuration.
 func (e *Engine) LoadConfigurations() error {
 	// Read every strategy files
+	errs := []error{}
+
 	for _, cfgFile := range GetFiles(e.Options.File) {
 
 		loadedConfiguration, err := config.New(cfgFile, e.Options.ValuesFiles, e.Options.SecretsFiles)
 
-		if err != nil && err != config.ErrConfigFileTypeNotSupported {
-			logrus.Errorf("%s\n\n", err)
+		switch {
+		case err == config.ErrConfigFileTypeNotSupported:
+			// Updatecli accepts either a single configuration file or a directory containing multiple configurations.
+			// When browsing files from a directory, we don't want to record error due to unsupported files.
 			continue
-		} else if err == config.ErrConfigFileTypeNotSupported {
+		case err != nil && err != config.ErrConfigFileTypeNotSupported:
```

Comment:
```suggestion
		case config.ErrConfigFileTypeNotSupported:
```

Reply:
I didn't realized the "nil" case you were speaking about, it's clear now, thx!

Reply:
Yeah, it was there for a (really) good reason. I personnaly like the switch with type assertion: easier to read.

Reply:
@dduportal I implemented your suggestion